app/src/main/java/fr/neamar/kiss/MainActivity.java

```diff
@@ -578,7 +581,7 @@ public void onFavoriteButtonClicked(View favorite) {
             public void run() {
                 result.fastLaunch(MainActivity.this);
             }
-        }, KissApplication.TOUCH_DELAY);
+        }, KissApplication.TOUCH_DELAY_FAVORITE);
```

Comment:
I like this fix, however if we use 0, I believe it will be even better to just call fastLaunch() directly without deferring this to another handler. It should be even faster (no threading) and make the code easier to read